### PR TITLE
Labels

### PR DESCRIPTION
**Pull request by SylvainCorlay**
@dmadeka you can reset #217 on this instead of doing a rebase.

### CODE REVIEW

File: js/src/LabelModel.js [hunk 1]
```diff
@@ -23,22 +23,119 @@ var LabelModel = markmodel.MarkModel.extend({
...
+        text: [],
         font_size: "14px",
         font_weight: "bold",
         align: "start"
-    })
+    }),
+
+    initialize: function() {
+        // TODO: Normally, color, opacity and size should not require a redraw
+        LabelModel.__super__.initialize.apply(this);
```
Comment (SylvainCorlay) [inline js/src/LabelModel.js]: now it will be important to call `initialize.apply(this, arguments)`

**Comment by dmadeka** [comment]
@SylvainCorlay Thanks a lot for this. I tried the rebase yesterday and ran out of non sleeping hours haha. I will reset to this, thanks.

**Comment by SylvainCorlay** [comment]
:+1:

**Comment by dmadeka** [comment]
Thanks @SylvainCorlay, all rebased. Working on the labels. Hoping to have a WordCloud.

**Pull request closed without merging**